main.mk

```diff
@@ -113,6 +113,13 @@ GOMPLATE ?= $(DOCKER) run \
 
 ECHO = @echo
 
+# For unit tests using
+BATS_REPORT = $(ROOT_DIR)/../../$(BATS_BIN_PATH) \
+	--report-formatter junit \
+	--formatter pretty -o $(ROOT_DIR)/../../ $(ROOT_DIR)/../../tests/*.bats
+BATS        = $(ROOT_DIR)/../../$(BATS_BIN_PATH) \
```

Comment:
I would only assign BATS to a binary that executes bats. The actual command I would name as 

CMD_RUB_BATS_TESTS or similar. Let me know what you think.